core/spec/models/spree/order_inventory_spec.rb

```diff
@@ -35,9 +35,18 @@
     end
 
     context "order is not completed" do
-      before { order.update_columns completed_at: nil }
+      let(:inventory_unit_finalizer) { double(:inventory_unit_finalizer, run!: [true]) }
+
+      before do
+        allow(Spree::Stock::InventoryUnitsFinalizer)
+          .to receive(:new).and_return(inventory_unit_finalizer)
+
+        order.update_columns completed_at: nil
+      end
 
       it "doesn't unstock items" do
+        expect(inventory_unit_finalizer).to_not receive(:run!)
```

Comment:
I have the feeling we can avoid this expectation: if `run!` is called the next expectation will fail, right? Is there any other reason you wanted to set this expectation?